bin/pycbc_multi_inspiral

```diff
@@ -381,353 +165,359 @@ with ctx:
+        'nifo': None,
+        'my_network_chisq': None, 
+        'reweighted_snr': None
+        }
     network_names = sorted(network_out_vals.keys())
-
-    event_mgr = events.EventManagerCoherent(
-        opt, opt.instruments, ifo_names, [ifo_out_types[n] for n in ifo_names],
-        network_names, [network_out_types[n] for n in network_names]
-        )
-
+    event_mgr = EventManagerCoherent(
+        args, args.instruments, ifo_names,
+        [ifo_out_types[n] for n in ifo_names], network_names,
+        [network_out_types[n] for n in network_names])
     logging.info("Read in template bank")
-    bank = waveform.FilterBank(opt.bank_file, flen, delta_f, complex64,
-                               low_frequency_cutoff=flow, phase_order=opt.order,
-                               taper=opt.taper_template,
-                               approximant=opt.approximant, out=template_mem)
-
+    bank = waveform.FilterBank(
+        args.bank_file, flen, delta_f, complex64, low_frequency_cutoff=flow,
+        phase_order=args.order, taper=args.taper_template,
+        approximant=args.approximant, out=template_mem)
     # Use injfilterrejector to reduce the bank to only those templates that
     # might actually find something
-    ntemplates = len(bank)
+    n_bank = len(bank)
     nfilters = 0
-
-    logging.info("Full template bank size: %s", ntemplates)
-    for ifo in opt.instruments:
+    logging.info("Full template bank size: %d", n_bank)
+    for ifo in args.instruments:
         bank.template_thinning(inj_filter_rejector[ifo])
-    if not len(bank) == ntemplates:
-        logging.info("Template bank size after thinning: %s", len(bank))
-
-    Fp = {}  # Antenna patterns
-    Fc = {}
-    for ifo in opt.instruments:
-        Fp[ifo], Fc[ifo] = pycbc.detector.Detector(ifo).antenna_pattern(
-            opt.ra, opt.dec, polarization=0, t_gps=t_gps)
-
-    for t_num, template in enumerate(bank):  # Loop over templates
-        for ifo in opt.instruments:
-            if opt.cluster_method == "window":
-                cluster_window = int(opt.cluster_window * sample_rate)
-            elif opt.cluster_method == "template":
+    if not len(bank) == n_bank:
+        n_bank = len(bank)
+        logging.info("Template bank size after thinning: %d", n_bank)
+    # Antenna patterns
+    fp = {}
+    fc = {}
+    for ifo in args.instruments:
+        fp[ifo], fc[ifo] = detector.Detector(ifo).antenna_pattern(
+            args.ra, args.dec, polarization=0, t_gps=t_gps)
+    # Loop over templates
+    for t_num, template in enumerate(bank):
+        for ifo in args.instruments:
+            if args.cluster_method == "window":
+                cluster_window = int(args.cluster_window * sample_rate)
+            elif args.cluster_method == "template":
                 cluster_window = int(template.chirp_length * sample_rate)
-            elif opt.cluster_window == 0:
+            elif args.cluster_window == 0:
                  cluster_window = int(0)
```

Comment:
Am I missing the point of having this if-else block inside the two for loops? It looks like you could easily move it to just before the loops.

Reply:
Ah right, it does depend on the template, but can you at least move it before the loop over the instruments?

Reply:
Actually, if you only use `cluster_window` in `cluster_template_network_events()`, then it is probably more readable to move this entire if-else block right before that call.

Reply:
Also, `int(0)` is equivalent to just `0`.

Reply:
Well spotted. I'll move this down and remove the last check as it's redundant.